src/controllers/router-store/test.tsx

```diff
@@ -49,10 +66,18 @@ describe('RouterStore', () => {
       const push = jest.spyOn(history, 'push');
       const replace = jest.spyOn(history, 'replace');
 
+      const plugins =
+        props.plugins ||
+        createPlugins({
+          context: props.resourceContext,
+          resourceData: props.resourceData,
+        });
+
       mount(
         <RouterContainer
           history={history}
           isGlobal
+          plugins={plugins}
```

Comment:
The fact that we have a `plugins` API that is `[plugin, plugin]` and this `plugins` that is the result of `combine([plugin, plugin])` caused me few headaches while reviewing. My brain was expecting `plugins` props to be the same, not change in some places but not in others.

So what about 
- we use `combine` only inside the bootstrap action. So the props/args called `plugins` are always arrays of plugin instances. 
- We can pick a different name for the value in the router store (eg `combinedPlugins`) so it is more obvious that it is no longer the array of plugins.

The result is also that these tests don't need to know about `combine` at all